### ISSUE TITLE
ImportError: No module named text when executing conan create (ubuntu installer)

### ISSUE THREAD

**Issue by L-M-V** [conan-io/conan#1868]
Hello,

New user to Conan here. I am running a pretty new linux mint installation and downloaded the ubuntu installer from the website. I'm following the steps in http://conanio.readthedocs.io/en/latest/packaging/getting_started.html

I am getting the following error:

```
~/test $ conan new Hello/0.1 -t
File saved: conanfile.py
File saved: test_package/CMakeLists.txt
File saved: test_package/conanfile.py
File saved: test_package/example.cpp
~/test $ conan create demo/testing
Hello/0.1@demo/testing: Exporting package recipe
Traceback (most recent call last):
  File "conan/conans/client/command.py", line 895, in run
  File "conan/conans/client/command.py", line 184, in create
  File "conan/conans/client/conan_api.py", line 63, in wrapper
  File "conan/conans/client/conan_api.py", line 292, in create
  File "conan/conans/client/manager.py", line 166, in export
  File "conan/conans/client/linter.py", line 21, in conan_linter
  File "conan/conans/client/linter.py", line 68, in _lint_py3
  File "conan/conans/client/linter.py", line 51, in _runner
  File "pylint/lint.py", line 1220, in __init__
  File "pylint/lint.py", line 458, in load_default_plugins
  File "pylint/lint.py", line 478, in _load_reporter
  File "astroid/modutils.py", line 437, in get_module_part
  File "astroid/modutils.py", line 338, in file_from_modpath
  File "astroid/modutils.py", line 383, in file_info_from_modpath
  File "astroid/modutils.py", line 603, in _spec_from_modpath
  File "astroid/interpreter/_import/spec.py", line 279, in find_spec
  File "astroid/interpreter/_import/spec.py", line 246, in _find_spec_with_path
ImportError: No module named text

ERROR: No module named text
```

As you can see above commands `conan` and `conan new` work fine. However `conan create` and `conan export` do not. 

Please let me know if you need more information.

**Comment by memsharded** [comment]
Yes, it seems a problem with the conan installer itself, not conan, that is using ``pyinstaller`` to create the executable, and for some reason it is not packaging correctly the required "text" module

Please try to use the recommended pip installation, so install python if not installed (it should be installed by default), and just:

```bash
$ pip install conan
```

We'll have a look to the installer, but if possible, please try the above, as the installers also have some other minor issues. Thanks for reporting!

**Comment by L-M-V** [comment]
Just thought I'd add my experience with the pip install command.

So pip wasn't installed by default in linux mint so I installed it and then Conan which went fine. However Conan wasn't added to my path or bin. This was confusing as a completely new user to pip/Conan, I searched for documentation about how to add it or call it directly and found little online. However it all worked fine after a system restart (I assume a logout / shell refresh would do similar). I assume this is pip's fault but as a suggestion, can you add a line to the installation page recommending a restart if conan isn't found.

**Comment by memsharded** [comment]
Yes, absolutely. It might be some different behavior of the Mint distro, typically the path is changed, so after a ``pip install`` you can already use the tool. I will add a couple of lines to the doc right now.

**Comment by memsharded** [comment]
I have been able to reproduce this, in a clean Ubuntu 16 installation

**Comment by memsharded** [comment]
Released 0.28 that solves this issue.
